www/src/components/guidelines/containers.js

```diff
@@ -1,39 +1,65 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import styled from "@emotion/styled"
-import propTypes from "@styled-system/prop-types"
+import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
-import { Box } from "./system"
-
-export const Container = styled(Box)()
-
-Container.defaultProps = {
+const container = {
+  boxSizing: `border-box`,
```

Comment:
Curious why you explicitly added this rule — did you not have `border-box` here without it? It should be set for all elements by default via the `*` selector: 
![image](https://user-images.githubusercontent.com/21834/82896387-2bdb0800-9f56-11ea-9cb1-43c66ec9f254.png)

Reply:
Earlier we were using `import { Box } from "./system"`, and this Box which we were importing was explicitly adding this rule,I thought there must be some reason behind it, so I kept it untouched. I will remove it now as it is getting set by `*` selector.
Thank you very much

Reply:
Ahh OK! 👍 

Most probably a leftover from the prototype then, which probably didn't use https://www.gatsbyjs.org/packages/gatsby-plugin-typography/, which normalizes `box-sizing` for us in the case of .org.